pontoon/base/models.py

```diff
@@ -966,6 +1070,9 @@ def for_project_locale(self, project, locale, paths=None, search=None):
                 pass
             entities = entities.filter(resource__path__in=paths) or entities
 
+        if filter_search:
+            entities = entities.filter(Q(**{'string__icontains': filter_search}) | Q(**{'string_plural__icontains': filter_search}))
+
```

Comment:
Instead of two (`search` & `filter_search`), we should only have one search parameter, which should search by:
* original strings (singular and plural)
* translations
* keys
* comments